src/snowflake/cli/plugins/nativeapp/version/version_processor.py

```diff
@@ -64,12 +65,14 @@ def check_index_changes_in_git_repo(
         # Check if the repo has any changes, including untracked files
         if repo.is_dirty(untracked_files=True):
             cc.warning(
-                "Changes detected in the git repository. (Rerun your command with --skip-git-check flag to ignore this check)"
+                "Changes detected in the git repository. "
+                "(Rerun your command with --skip-git-check flag to ignore this check)"
             )
             repo.git.execute(["git", "status"])
 
             user_prompt = (
-                "You have local changes in this repository that are not part of a previous commit. Do you still want to continue?",
```

Comment:
this was actually a tuple is disguise:
![image](https://github.com/snowflakedb/snowflake-cli/assets/171080004/793d76cb-247c-4275-851e-5926e867f65d)